cms/templates/organisms.html

```diff
@@ -0,0 +1,44 @@
+{% import 'molecules.html' as molecules %}
+{% import 'atoms.html' as atoms %}
+
+{% macro list_of_images_and_heading(view) -%}
+<ul>
+{% for cat in view.get_categories() %}
+    <li>
+    	<a href={{view.request.route_url('category_jinja2', category=cat.uuid)}}>
+    		{{molecules.image_and_heading(view, cat)}}
+       	</a>
+    </li>
+{% endfor %}
+</ul>
+{%- endmacro %}
+
+{% macro list_of_links(view, pages) -%}
```

Comment:
This is a bit misleading. `list of links` sounds like it should take a list and render links, instead it's expecting a page and use the `page.title`. I'd recommend either chaning the name or making this more generic by expenging a list of tuples or a dict.